misc/python/materialize/output_consistency/common/probability_settings.py

```diff
@@ -0,0 +1,18 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+
+class ProbabilitySettings:
```

Comment:
It seems a bit weird to me to have a class and object just to store some constants. If you call the file `probabilities.py` and import it with `import materialize.output_consistency.common.probabilities`, then you can just have a `HORIZONTAL_LAYOUT_WHEN_NOT_AGGREGATED = 0.9` constant at the top level, and use it in other files as `probabilities.HORIZONTAL_LAYOUT_WHEN_NOT_AGGREGATED`.

But maybe you have more planned with this and I'm missing something.

Reply:
You are right that these are currently constants and pretty likely to stay that (that is, they are not properties configurable at runtime). I will change it.